stubs/tensorflow/tensorflow/__init__.pyi

```diff
@@ -0,0 +1,68 @@
+# Alias for bool is used because tensorflow name shadows bool with tf.bool.
+from builtins import bool as _bool
+from typing import Any, Iterable, Iterator, NoReturn, overload
+
+import numpy as np
```

Comment:
How do I handle numpy errors? I thought requires line in METADATA.toml would allow using numpy?

numpy is only library I think needed as most tensorflow functions accept numpy arrays too.